test/functional/change_stream.test.js

```diff
@@ -1475,7 +1475,7 @@ describe('Change Streams', function() {
     }
   });
 
-  it('should resume piping of Change Streams when a resumable error is encountered', {
+  it.skip('should resume piping of Change Streams when a resumable error is encountered', {
```

Comment:
Is this a sometimes leaky test?

Reply:
It was leaking the `'_nodemongodbnative_resumepipe.txt'` resource at some point in my testing, but I forget exactly when/where. I'll unskip to check if it's still an issue.